pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -178,6 +178,19 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 	}
 
 	remaining := pp.totalResource.Clone()
+
+	// Process guarantee resources of the queues
+	// We must substract the guarantees from the remaining before
+	// fairness divison of the remianing
+	for _, attr := range pp.queueOpts {
+		// Queue can't desrve less resources than it has in guarantee
+		if attr.deserved.LessEqual(attr.guarantee, api.Zero) {
+			guarantee := attr.guarantee.Clone()
+			attr.deserved = guarantee
+			remaining.Sub(guarantee)
```

Comment:
The guaranted resources have already been subed here: https://github.com/volcano-sh/volcano/blob/79a8e41ebdb04b6e218deee04366f77b0a893ac1/pkg/scheduler/plugins/proportion/proportion.go#L120, do we still need to sub it ?